src/test_log/template_code.cpp

```diff
@@ -5,25 +5,25 @@
  * @author  Dylan Rakiman, Arsalan Anwari
  */
 
-#include <iostream>
+// #include <iostream>
```

Comment:
Can these comments be removed? If not, maybe move it into a function so users are still able to run this test if they like.